### ISSUE TITLE
GUPS -n parameter

### ISSUE THREAD

**Issue by davidozog** [Sandia-OpenSHMEM/SOS#430]
Looks like the -n parameter (number of updates per PE) is silently ignored.

**Comment by jdinan** [comment]
Should we just remove the parameter?  It doesn't appear to be part of the official benchmark specification (http://icl.cs.utk.edu/projectsfiles/hpcc/RandomAccess/).

**Comment by davidozog** [comment]
Yeah, probably.  I don't see any reference to multiple updates in that link either.

**Comment by jdinan** [comment]
I think our version originated here https://github.com/openshmem-org/gups-shmem, which in turn came from here http://icl.cs.utk.edu/hpcc/software/index.html, in case you're looking for breadcrumbs.

**Comment by davidozog** [comment]
I had forgotten about this, but it's come up again.  On second thought, I actually recommend we implement the parameter instead of removing it.  The GUPS measurement can be extremely noisy with the default value (which is arbitrarily 4 times the number of table elements).  By adding the `-n` parameter, which is pretty trivial, it's convenient to increase the parameter where I see far less variation between executions, at the cost of just waiting a few more seconds.
